jenkins/plugins.txt

```diff
@@ -52,7 +52,7 @@ echarts-api:5.6.0-5
 extended-read-permission:61.vf24570ff3b_e9
 favorite:2.237.v79163ca_8b_892
 font-awesome-api:6.7.2-1
-gatling:136.vb_9009b_3d33a_e
+gatling:1.3.0
```

Comment:
FWIW this is fixed on the master branch it just hasn't been released 😢.

Reply:
Coolio, the docs advise downgrading for now. Do you think it'll get released in the next few days?

Reply:
I replied on a random GitHub PR to ask him to do a release.

there's no harm in downgrading for now but set a reminder for a week or two to check or something